src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/PolldaddyWidgetPageObject.java

```diff
@@ -0,0 +1,95 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.widget;
+
+import org.apache.commons.lang.NotImplementedException;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+/**
+ * @ownership: Content X-Wing
+ */
+public class PolldaddyWidgetPageObject extends WidgetPageObject {
+
+  @FindBy(css = ".PDS_Poll")
+  private List<WebElement> polldaddyDivList;
+  By polldaddyBody = By.cssSelector("*");
+
+  private static final String TAG_NAME = "pollydaddy";
+  private static final String ARTICLE_NAME = "PolldaddyWidget";
+  private static final String TAG =
+      "<polldaddy id=\"8956579\"/>";
+  private static final String[] TAGS = {
+      "<polldaddy id=\"8956579\"/>",
+      "<polldaddy id=\"9022741\"/>"
+  };
+  private static final String INCORRECT_TAG = "<polldaddy />";
+  private static final String ERROR_MESSAGE =
+      "Failed to render the Polldaddy widget. Please check if all required parameters are in place.";
+
+  public PolldaddyWidgetPageObject(WebDriver driver) {
+    super(driver);
+  }
+
+  protected String getArticleName() {
+    return ARTICLE_NAME;
+  }
+
+  protected String getTagName() {
+    return TAG_NAME;
+  }
+
+  public String getTag() {
+    return TAG;
+  }
+
+  @Override
```

Comment:
Why are you using `@Override` here? Isn't just an implementation of [the abstract method](https://github.com/Wikia/selenium-tests/blob/XW-350/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/WidgetPageObject.java#L35)?